test/runtests.jl

```diff
@@ -9,9 +9,9 @@ dat = readcbfdata("example4.cbf")
 
 c, A, b, con_cones, var_cones, vartypes, dat.sense, dat.objoffset = cbftompb(dat)
 
-@test_approx_eq c [1.0, 0.64]
-@test_approx_eq A [-50.0 -31; -3.0 2.0]
-@test_approx_eq b [-250.0, 4.0]
+@test c ≈ [1.0, 0.64]
```

Comment:
This only works on 0.6. You can use `@test isapprox(...)` instead.

Reply:
It works on 0.6+ right? Are people who would use CBU still using 0.5? We aren't for the paper.

Reply:
Yes, you can update the package to 0.6+ if you'd like. Please keep the commits separate.

Reply:
is that OK?